src/net/textproto/reader.go

```diff
@@ -481,6 +481,15 @@ func (r *Reader) ReadDotLines() ([]string, error) {
 //	}
 //
 func (r *Reader) ReadMIMEHeader() (MIMEHeader, error) {
+	header, _, err := r.readMIMEHeader(false /* recordRequestLines */)
+	return header, err
+}
+
+func (r *Reader) CFReadMIMEHeader(recordRequestLines bool) (MIMEHeader, []CFHeaderLine, error) {
+	return r.readMIMEHeader(recordRequestLines)
```

Comment:
This function can be removed now and the other function below can be renamed to `CFReadMIMEHeader` right?

Reply:
Done.